values-schema.yaml

```diff
@@ -797,7 +797,7 @@ definitions:
       - name
       - type
   backupSchedule:
-    default: 0 0 * * *
+    default: 0 0 0 * * *
```

Comment:
I am not sure why this is changed, but it does not look right to me (comparing with https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#schedule-syntax). But we might want to remove this line and move it to defaults. If we make any change here, a regex validation could come in useful. But it's not really related to the purpose of this PR.

Reply:
Confusing docs there. They say that it follows the Kubernetes CronJobs expression (which is 5 digits) in the description, but the link leads to Go-library docs. I would suggest we stick to the 5-digit format and update the templates to set the first digit to `0`, for the following reasons:
* Many customers who already have used this might have this wrong written to their values. This is tricky to migrate with the methods we currently have.
* We state "Cron-type expression" in the field description. This is misleading without explicitly referencing these Go-library docs.
* We use CronJobs in other places (e.g. Gitea backup, Kyverno) and it would be inconsistent.
* There is no added value in adding second-precision. There is no practical use for it.

Reply:
Updated it to use 5digit cron definitions. The additional 0 will be added in the otomi-db cluster definition.